### PR TITLE
fixed issues with inconsistent whitespace and line wrapping in help

### PR DESCRIPTION
**Pull request by GodloveD**
**Description of the Pull Request (PR):**

For a while now the docs have been inconsistently indented (long descriptions and examples on many subcommands are indented further than other help) and the line wrapping has not been done consistently at 80 columns.  This is not a huge deal, but it's annoying.

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: docs/content.go [hunk 1]
```diff
@@ -831,8 +843,8 @@ Enterprise Performance Computing (EPC)`
 	OciUpdateUse   string = `update [update options...] <container_ID>`
 	OciUpdateShort string = `Update container cgroups resources (root user only)`
 	OciUpdateLong  string = `
-  Update will update cgroups resources for the specified container ID.
-  Container must be in a RUNNING or CREATED state.`
+  Update will update cgroups resources for the specified container ID. Container 
+  smust be in a RUNNING or CREATED state.`
```
Comment (WestleyK) [inline docs/content.go]: What is `smust`?

File: docs/content.go [hunk 2]
```diff
@@ -850,13 +862,15 @@ Enterprise Performance Computing (EPC)`
...
+  Resume will resume all processes previously paused for the specified container 
+  ID.`
 	OciResumeExample string = `
   $ singularity oci resume mycontainer`
 
 	OciMountUse   string = `mount <sif_image> <bundle_path>`
-	OciMountShort string = `Mount create an OCI bundle from SIF image (root user only)`
-	OciMountLong  string = `
+	OciMountShort string = `Mount create an OCI bundle from SIF image (root user 
+  only)`
```
Comment (WestleyK) [inline docs/content.go]: Do we really need `(root user only)` for every oci command?

File: docs/plugin.go [hunk 3]
```diff
@@ -37,74 +37,74 @@ const (
 	PluginCompileUse   string = `compile [compile options...] <host_path>`
 	PluginCompileShort string = `Compile a singularity plugin`
 	PluginCompileLong  string = `
-	The 'plugin compile' command allows a developer to compile a singularity plugin in the
-	expected environment. The provided host directory is the location of the plugin's
-	source code.`
+The 'plugin compile' command allows a developer to compile a singularity plugin 
+in the expected environment. The provided host directory is the location of the 
+plugin's source code.`
```
Comment (WestleyK) [inline docs/plugin.go]: Can you put the 2 spaces here? To be consistent with all the other helps:

```suggestion
  plugin's source code.`
```

File: docs/plugin.go [hunk 4]
```diff
@@ -37,74 +37,74 @@ const (
...
 	PluginCompileExample string = `
-	$ singularity plugin compile $HOST_PATH`
+$ singularity plugin compile $HOST_PATH`
 
 	// // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	// // plugin install command
 	// // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (WestleyK) [inline docs/plugin.go]:
```suggestion
	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: docs/plugin.go [hunk 5]
```diff
@@ -37,74 +37,74 @@ const (
...
 	PluginInstallUse   string = `install [install options...] <plugin_path>`
 	PluginInstallShort string = `Install a singularity plugin`
 	PluginInstallLong  string = `
-	The 'plugin install' command installs the plugin found at plugin_path into the
-	appropriate directory on the host.`
+The 'plugin install' command installs the plugin found at plugin_path into the
+appropriate directory on the host.`
```
Comment (WestleyK) [inline docs/plugin.go]: Here too

File: docs/plugin.go [hunk 6]
```diff
@@ -37,74 +37,74 @@ const (
...
 	PluginInspectLong  string = `
-	The 'plugin inspect' command allows a user to inspect a plugin that is already
-	installed in the system or an image containing a plugin that is yet to be installed.`
+The 'plugin inspect' command allows a user to inspect a plugin that is already
+installed in the system or an image containing a plugin that is yet to be installed.`
 	PluginInspectExample string = `
-	$ singularity plugin inspect sylabs.io/test-plugin
-	$ singularity plugin inspect test-plugin.sif`
+$ singularity plugin inspect sylabs.io/test-plugin
+$ singularity plugin inspect test-plugin.sif`
```
Comment (WestleyK) [inline docs/plugin.go]: Here too

File: docs/remote.go [hunk 7]
```diff
@@ -13,11 +13,11 @@ const (
 	RemoteUse   string = `remote <subcommand>`
 	RemoteShort string = `Manage singularity remote endpoints`
 	RemoteLong  string = `
-	The 'remote' commands allow you to manage singularity remote endpoints through its
-	subcommands. These allow you to add, log in, and use endpoints. The remote
-	configuration is stored in $HOME/.singularity/remotes.yaml by default.`
+The 'remote' commands allow you to manage singularity remote endpoints through 
+its subcommands. These allow you to add, log in, and use endpoints. The remote
+configuration is stored in $HOME/.singularity/remotes.yaml by default.`
```
Comment (WestleyK) [inline docs/remote.go]: Here too

```suggestion
  configuration is stored in $HOME/.singularity/remotes.yaml by default.`
```

File: docs/remote.go [hunk 8]
```diff
@@ -27,61 +27,61 @@ const (
...
 	RemoteStatusLong  string = `
-	the 'remote status' command checks the status of the specified remote endpoint
-	and reports the availibility of services and their versions. If no endpoint is
-	specified, it will check the status of the default remote.`
+The 'remote status' command checks the status of the specified remote endpoint
+and reports the availibility of services and their versions. If no endpoint is
+specified, it will check the status of the default remote.`
 	RemoteStatusExample string = `
-	$ singularity remote status SylabsCloud`
+$ singularity remote status SylabsCloud`
```
Comment (WestleyK) [inline docs/remote.go]: Here too, for this whole file

```suggestion
  $ singularity remote status SylabsCloud`
```

**Review by WestleyK — CHANGES_REQUESTED** [review]
Looks good, couple suggestions...

File: docs/content.go (hunk 2 above)
Comment (GodloveD) [reply docs/content.go]: Yeah.  I think we do.  It looks redundant in the help doc, but remember that only one of these commands will be displayed with any help query.

**Comment by GodloveD** [comment]
Nice catches @WestleyK.  Thanks for keeping me honest.  I think it should be good to go now.

**Review by WestleyK — APPROVED** [review]
Looks good to me :+1:

File: docs/content.go [hunk 9]
```diff
@@ -162,7 +162,8 @@ Enterprise Performance Computing (EPC)`
 	CacheUse   string = `cache`
 	CacheShort string = `Manage the local cache`
 	CacheLong  string = `
-  Manage your local singularity cache. You can list/clean using the specific types.`
+  Manage your local singularity cache. You can list/clean using the specific
```
Comment (GwendolynK-zz) [inline docs/content.go]:
```suggestion
  Manage your local Singularity cache. You can list/clean using the specific 
```

File: docs/plugin.go [hunk 10]
```diff
@@ -37,74 +37,75 @@ const (
 	PluginCompileUse   string = `compile [compile options...] <host_path>`
 	PluginCompileShort string = `Compile a singularity plugin`
 	PluginCompileLong  string = `
-	The 'plugin compile' command allows a developer to compile a singularity plugin in the
-	expected environment. The provided host directory is the location of the plugin's
-	source code.`
+  The 'plugin compile' command allows a developer to compile a singularity
```
Comment (GwendolynK-zz) [inline docs/plugin.go]:
```suggestion
  The 'plugin compile' command allows a developer to compile a Singularity 
```

File: docs/plugin.go [hunk 11]
```diff
@@ -37,74 +37,75 @@ const (
...
-	// // plugin list command
-	// // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	// plugin list command
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	PluginListUse   string = `list [list options...]`
 	PluginListShort string = `List installed singularity plugins`
 	PluginListLong  string = `
-	The 'plugin list' command lists the singularity plugins installed on the host.`
+  The 'plugin list' command lists the singularity plugins installed on the host.`
```
Comment (GwendolynK-zz) [inline docs/plugin.go]:
```suggestion
  The 'plugin list' command lists the Singularity plugins installed on the host.`
```

File: docs/remote.go [hunk 12]
```diff
@@ -13,11 +13,11 @@ const (
 	RemoteUse   string = `remote <subcommand>`
 	RemoteShort string = `Manage singularity remote endpoints`
 	RemoteLong  string = `
-	The 'remote' commands allow you to manage singularity remote endpoints through its
-	subcommands. These allow you to add, log in, and use endpoints. The remote
-	configuration is stored in $HOME/.singularity/remote.yaml by default.`
+  The 'remote' commands allow you to manage singularity remote endpoints through
```
Comment (GwendolynK-zz) [inline docs/remote.go]:
```suggestion
  The 'remote' commands allow you to manage Singularity remote endpoints through 
```

File: docs/remote.go [hunk 13]
```diff
@@ -27,61 +27,61 @@ const (
 	RemoteAddUse   string = `add <remote_name> <remote_URI>`
 	RemoteAddShort string = `Create a new singularity remote endpoint`
 	RemoteAddLong  string = `
-	The 'remote add' command allows you to create a new remote endpoint to be
-	be used for singularity remote services.`
+  The 'remote add' command allows you to create a new remote endpoint to be used 
+  for singularity remote services.`
```
Comment (GwendolynK-zz) [inline docs/remote.go]:
```suggestion
  for Singularity remote services.`
```

File: docs/remote.go [hunk 14]
```diff
@@ -27,61 +27,61 @@ const (
...
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	// remote use command
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	RemoteUseUse   string = `use <remote_name>`
 	RemoteUseShort string = `Set a singularity remote endpoint to be actively used`
 	RemoteUseLong  string = `
-	The 'remote use' command sets the remote to be used by default by any command
-	that interacts with singularity services.`
+  The 'remote use' command sets the remote to be used by default by any command
+  that interacts with singularity services.`
```
Comment (GwendolynK-zz) [inline docs/remote.go]:
```suggestion
  that interacts with Singularity services.`
```

File: docs/content.go [hunk 15]
```diff
@@ -363,9 +367,11 @@ Enterprise Performance Computing (EPC)`
 	CapabilityDropUse   string = `drop [drop options...] <capabilities>`
 	CapabilityDropShort string = `Remove capabilities from a user or group (requires root)`
 	CapabilityDropLong  string = `
-  Remove Linux capabilities from an user/group. NOTE: This command requires root to run.
+  Remove Linux capabilities from an user/group. NOTE: This command requires root
```
Comment (GwendolynK-zz) [inline docs/content.go]:
```suggestion
  Remove Linux capabilities from a user/group. NOTE: This command requires root 
```
Even though 'user' starts with a vowel, this 'u' is actually a glide/semivowel, meaning it is treated as a consonant.

**Review by GwendolynK-zz — CHANGES_REQUESTED** [review]

File: docs/content.go (hunk 9 above)
Comment (GodloveD) [reply docs/content.go]: I think in general we want to capitalize Singularity when we are referring to the container platform and lower case it when we are referring to the command (as in `singularity`).  At least that is how I've written docs and slides and stuff.

In this context we are referring to the platform not the command, so I believe uppercase is the way to go.

**Comment by GodloveD** [comment]
Thanks for the suggestions @GwendolynK.  I accepted them all.  I think this should be good to merge now.

**Review by GwendolynK-zz — APPROVED** [review]

**Pull request merged**